package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "install": "node install.js",
     "test": "tape test/*.js",
-    "cache-clean": "rimraf ~/.electron"
```

Comment:
the typo is referring to an actually typo from previous commit before where I wrote 'rm -f ~/.electron` by mistake. But also `rimraf` isn't in the package.json and `~/` isn't cross platform either :P 

The scripts in this PR are decent (assuming mac/linux) https://github.com/mafintosh/electron-prebuilt/pull/30